exercises/twelve-days/src/test/java/TwelveDaysTest.java

```diff
@@ -0,0 +1,172 @@
+                "and a Partridge in a Pear Tree.\n";
+        assertEquals(expectedVerseSix, TwelveDays.verse(6));
+    }
+
+    @Ignore
+    @Test
+    public void testVerseSeven() {
+        String expectedVerseSeven = "On the seventh day of Christmas my true love gave to me, " +
+                "seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, " +
+                "two Turtle Doves, and a Partridge in a Pear Tree.\n";
+        assertEquals(expectedVerseSeven, TwelveDays.verse(7));
+    }
+
+    @Ignore
+    @Test
+    public void testVerseEight() {
+        String expectedVerseEight = "On the eighth day of Christmas my true love gave to me, eight Maids-a-Milking," +
+                " seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, " +
+                "three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n";
+        assertEquals(expectedVerseEight, TwelveDays.verse(8));
+    }
+
+    @Ignore
+    @Test
+    public void testVerseNine() {
+        String expectedVerseNine = "On the ninth day of Christmas my true love gave to me, nine Ladies Dancing, " +
+                "eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, " +
+                "four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n";
+        assertEquals(expectedVerseNine, TwelveDays.verse(9));
+    }
+
+    @Ignore
+    @Test
+    public void testVerseTen() {
+        String expectedVerseTen = "On the tenth day of Christmas my true love gave to me, ten Lords-a-Leaping, " +
+                "nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, " +
+                "five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, " +
+                "and a Partridge in a Pear Tree.\n";
+        assertEquals(expectedVerseTen, TwelveDays.verse(10));
+    }
+
+    @Ignore
+    @Test
+    public void testVerseEleven() {
+        String expectedVerseEleven = "On the eleventh day of Christmas my true love gave to me, " +
+                "eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, " +
+                "seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, " +
+                "three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n";
+        assertEquals(expectedVerseEleven, TwelveDays.verse(11));
+    }
+
+    @Ignore
+    @Test
+    public void testVerseTwelve() {
+        String expectedVerseTwelve = "On the twelfth day of Christmas my true love gave to me, " +
+                "twelve Drummers Drumming, eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, " +
+                "eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, " +
+                "four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n";
```

Comment:
I didn't have the newline there originally, but I changed it when I saw that the exercise in other languages had the newline at the end of every verse. I think it's because if you copy a line of the song from the description, you get the newline as well.